ConfigurationValidation.hpp

```diff
@@ -303,11 +303,15 @@
 //////////////////////////////////////////
 
 #if (DEC_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART)
-  #if (DEC_MOTOR_CURRENT_RATING > 1700)
-    #error "The TMC2209 driver is only rated up to 1.7A output. Delete this error if you know what youre doing" 
-  #endif
-  #if (DEC_MOTOR_CURRENT_RATING == 0)
-    #error "DEC current rating/setting is not configured. Enter the current rating of your motor" 
+  #if defined(DEC_MOTOR_CURRENT_RATING)
+    #if (DEC_MOTOR_CURRENT_RATING > 1700)
+      #error "The TMC2209 driver is only rated up to 1.7A output. Delete this error if you know what youre doing" 
+    #endif
+    #if (DEC_MOTOR_CURRENT_RATING == 0)
+      #error "DEC current rating/setting is not configured. Enter the current rating of your motor" 
```

Comment:
This message is not going to allow users to recover easily. The second sentence should read something like:

Please define the current rating of your motor in you local configuration file using the DEC_MOTOR_CURRENT_RATING keyword.

Reply:
Same for line 314

Reply:
Fixed